src/lms-ds-loader/edfi_lms_ds_loader/scripts/mssql/create_processed_files_table.sql

```diff
@@ -0,0 +1,21 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Licensed to the Ed-Fi Alliance under one or more agreements.
+ * The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+ * See the LICENSE and NOTICES files in the project root for more information.
+ */
+
+CREATE TABLE lms.ProcessedFiles (
+    FullPath NVARCHAR(255) NOT NULL,
```

Comment:
I was a little concerned if this is a good idea - having the `FullPath` as the primary key. But I think it is the right decision:

1. It is truly unique, so if not primary key then need a separate unique index.
2. We have no reason to think that this table will ever be a foreign key for another table. If that ever happens, then we should introduce an identity column instead.
3. And we don't anticipate updating existing rows, just writing new ones.